### ISSUE TITLE
Provide a default implementation for Dashboard when analytics are not setup

### ISSUE THREAD

**Issue by elrayle** [samvera/hyrax#5275]
### Descriptive summary

When neither google analytics or mamota are configured in `config/analytics.yml`, the Dashboard gets an error.  It would be helpful to have a default implementation for the Dashboard that displays a message like...

> Analytics are not configured.  To display analytics on the Dashboard, setup either Google Analytics or Matoma and configure access in /config/analytics.yml.

### Rationale

Displaying errors in the UI should be avoided by providing default behavior.

### Expected behavior

Either some kind of simple analytics out-of-the-box, or at least some kind of message on the Dashboard page.

### Actual behavior

http://localhost:3000/dashboard produces error...

![image](https://user-images.githubusercontent.com/6855473/145074782-1202687e-1f8c-419c-9c0a-270eded12c22.png)

### Steps to reproduce the behavior

Prereq:
* do not include any ENV values for the variables in `/config/analytics.yml`

To reproduce:
1. Login
2. navigate to Dashboard

Error will display, but prefer a page created without erroring out that includes with informative message.

### Related work

PR #5089 - Analytics

**Comment by sara-g** [comment]
@elrayle  In this scenario, HYRAX_ANALYTICS is set to true but no analytics is configured?   If HYRAX_ANALYTICS is off, then you should get a user friendly message when clicking on analytics reports and dashboard should load.

**Comment by elrayle** [comment]
We worked on this in slack.  Results of the investigation:

* There is a default implementation when `Hyrax.config.analytics? => false`
* `Hyrax.config.analytics?` is set through `ENV['HYRAX_ANALYTICS'] and defaults to `false` if that variable isn't set.  Except the ENV variable won't be checked if `@analytics` is already set, in which case it returns the value of `@analytics`
* The value `@analytics` can be set through `Hyrax.config.analytics=`.  This was the old way of turning analytics on/off.  In older analytics, this was set via `initializers/hyrax.rb`.  This setting should be removed from the initializer.

The final comment is what was causing the problem.  Once that was removed, this revealed that there is a default implementation.  

The removal of `config.analytics=true` from the initializer needs to be added to release notes.

Thanks to @sara-g for the help tracking this down.

Closing the issue.
